.ci/s390x/clean_up_s390x.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 IBM
```

Comment:
2019 (here and below)?

Reply:
Thank you, i just copied them and forgot to change the year